functorch/experimental/_cond.py

```diff
@@ -40,8 +40,10 @@ def trace_cond(proxy_mode, func_overload, pred, true_fn, false_fn, operands):
     assert isinstance(operands, (list, tuple)), "Cond operands must be a list or tuple of tensors"
     assert all(isinstance(o, torch.Tensor) for o in operands), "Cond operands must be a list of tensors"
 
-    true_graph = get_isolated_graphmodule(true_fn, operands, {})
-    false_graph = get_isolated_graphmodule(false_fn, operands, {})
+    flattened_inputs, spec = pytree.tree_flatten(operands)
```

Comment:
@tugsbayasgalan we don't need this line, since line 40 and 41 already guarantee you have a flattened list?

Reply:
Yeah you are right. Will update :)